public/index.html

```diff
@@ -24,20 +24,22 @@ <h1>Join</h1>
                     <label for="room">Choose a room:</label>
                     <input
                         list="roomsList"
-                        name="room"
                         id="room"
+                        name="room"
                         placeholder="Room"
                         required
                     />
                     <datalist id="roomsList"> </datalist>
+                    <input type="hidden" name="roomId" id="room-hidden">
 
                     <button>Join</button>
                 </form>
             </div>
         </div>
         <script id="roomsTemplate" type="text/html">
             {{#rooms}}
-            <option value="{{.}}">{{/rooms}}</option>
+            <option data-value="{{id}}">{{title}}</option>
```

Comment:
Because I need to pass both room name (for new rooms) and id (for existing rooms) to the backend (from the next page).
I also want to only show room name in the drop list, and I can't do that with `<option value=id>Room Name</option>`. Because this way the user will see both id and room name in the drop list, which I don't think is right.
(Firefox doesn't actually show the `value`, but Chrome does)